autobahn/client.py

```diff
@@ -16,15 +15,15 @@
 
 
 async def get_case_count(url):
-    url = URL(url).with_path('/getCaseCount')
+    url = url + '/getCaseCount'
     async with open_websocket_url(url) as conn:
         case_count = await conn.get_message()
         logger.info('Case count=%s', case_count)
     return int(case_count)
 
 
 async def run_case(url, case):
-    url = URL(url).with_path('/runCase').with_query(case=case, agent=AGENT)
+    url = url + '/runCase?case={}&agent={}'.format(case, AGENT)
```

Comment:
would you confirm that `with_query()` isn't providing some necessary escaping?

Reply:
The autobahn server was broken (preexisting), I pushed a commit to fix it.

After that I compared the server/client results before/after this PR and there are no non-spurious differences. Although there are many preexisting failures.

Reply:
oh, I didn't realize this was just just the autobahn test code

thank you